cmake/SetupDeps.cmake

```diff
@@ -64,9 +66,22 @@ if (HDF5_FOUND)
       else()
 	message("MPI::MPI_CXX provided by parent package")
       endif()
+    else()
+      message(status "HDF5 is not parallel")
```

Comment:
`message(STATUS`?

Reply:
case-senstivity is weird in CMake but I think in this case it's insensitive

Reply:
I briefly googled this out of curiosity and the first example was `message(status` and apparently it actually does matter in this case: https://stackoverflow.com/a/35034125

Reply:
It looks like that stack overflow is about the case of the thing the message is about, not the STATUS flag? The rest of the code is using lower-case, which is why I kept doing it